workspaces/pingidentity/catalog-info.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: pingidentity
+  description: An example of a Backstage application.
+  # Example for optional annotations
+  # annotations:
+  #   github.com/project-slug: backstage/backstage
+  #   backstage.io/techdocs-ref: dir:.
+spec:
+  type: website
+  owner: john@example.com
+  lifecycle: experimental
```

Comment:
This should be `production`

Reply:
@kim-tsao If I understand correctly we are reverting this back to `experimental`?

Reply:
@JessicaJHee yes, please revert as it was decided just today